third_party/terraform/resources/resource_google_folder_organization_policy.go

```diff
@@ -43,8 +43,8 @@ func resourceFolderOrgPolicyImporter(d *schema.ResourceData, meta interface{}) (
 	config := meta.(*Config)
 
 	if err := parseImportId([]string{
-		"folders/(?P<folder>[^/]+):constraints/(?P<constraint>[^/]+)",
```

Comment:
Do we need similar changes for the datasources?

Reply:
Fixed, thanks.